src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRental = 40;
+  const discountFromThreeDays = 20;
+  const discountFromSevenDays = 50;
+
+  let totalAmount = dailyRental * days;
+
+  if (days >= 7) {
+    totalAmount -= discountFromSevenDays;
+
+    return totalAmount;
```

Comment:
You can remove this return statement since there's already a return statement at the end of the function